### PR TITLE
LPS-184923 | master

### PR DESCRIPTION
**Pull request by JessieZhao0714**
https://liferay.atlassian.net/browse/LPS-184923

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/Audit.testcase [hunk 1]
```diff
@@ -46,6 +46,102 @@ definition {
...
+		task ("When: Get HTTP response headers in login state") {
+			var curl = '''http://localhost:8080/web/guest?SM_USER=test@liferay.com -I''';
+
+			var response = JSONCurlUtil.post(${curl});
+		}
+
+		task ("Then: Assert the values of X-Liferay-Request-Company and X-Liferay-Request-Group are companyId and groupId respectively and X-Liferay-Request-Guest-User is false.") {
+			TestUtils.assertPartialEquals(
+				actual = ${response},
+				expected = "X-Liferay-Request-Company: 20096");
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/Audit.testcase:64]: The companyId will change after a period of time, try not to write a fixed value, please use JSON to get values.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/Audit.testcase [hunk 2]
```diff
@@ -46,6 +46,102 @@ definition {
...
+
+			TestUtils.assertPartialEquals(
+				actual = ${response},
+				expected = "X-Liferay-Request-Group: 20119");
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/Audit.testcase:68]: The groupId will change after a period of time, try not to write a fixed value, please use JSON to get values.

**Review by JoyceWang08211 — CHANGES_REQUESTED** [review]

**Comment by JessieZhao0714** [comment]
ci:test:security

**Comment by JessieZhao0714** [comment]
ci:test:sf

**Review by gaborlovasdotcom — APPROVED** [review]

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**